src/components/map/MapDirective.js

```diff
@@ -79,6 +88,49 @@ goog.require('ga_permalink');
 
             map.setTarget(element[0]);
 
+            scope.$watch('::ol3d', function(ol3d) {
```

Comment:
The use of `=gaMapOl3d` creates a watcher. And you create another watcher by calling `$watch` here. So to remove both watchers when `ol3d`  gets a defined value in the user scope, two one-time binding expressions should be used.